### PR TITLE
finished Lab 2 and put it into correct branch

### PR DESCRIPTION
**Pull request by sierrawilde**
@Higgins00 here is my lab 2. I am not sure if I explained my work well enough on the last problem, so please advise. Thanks!

### CODE REVIEW

File: Lab2.ipynb [hunk 1]
```diff
@@ -0,0 +1,1105 @@
...
+   "metadata": {},
+   "source": [
+    "# Lab 2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Sierra Wilde"
```
Comment (Higgins00) [inline Lab2.ipynb]: Olivia asked us to put both our names here so it is easier for her to figure out partners and grade

File: Lab2.ipynb [hunk 2]
```diff
@@ -0,0 +1,1105 @@
...
+   "metadata": {},
+   "source": [
+    "I have chosen the average number of background, cosmic ray counts per day to be X = 0.75, and the average number of gamma rays emitted per day to be Y = 2.0."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The Poisson distribution is a discrete distribution that describes the probability of measuring a certain number of counts in an interval of time. The distriutions are described by:"
```
Comment (Higgins00) [inline Lab2.ipynb]: Typo at the end. Distributions

File: Lab2.ipynb [hunk 3]
```diff
@@ -0,0 +1,1105 @@
...
...10x/5jM3B54R/n6QM2NR0yJiNkRMfuxxx7rVeCSJEnSYNTKykEXsFnF+5HAonrLRMSawAbA4vL9SOAHwAcz80/dK2Tmw+Xfp4DvUty+tILMnJqZ4zJz3CabbNInOyRJkiQNZK3synQWsGVEjAEeBiYC769aZgZwHHAHcATwi8zMiHgN8BPgc5n5/7oXLisQr8nMxyNiLeBg4Gct3Id+ze66JEmS1JdaVjnIzKURcRJFT0NDgGmZeX9EnAbMzswZwCXAFRExj6LFYGK5+knAFsAXI+KLZdq7gGeAm8uKwRCKisHFrdoHSVLvRcQw4Hbg1Mz8cUSsAZwOrE9R/l/WMANJ6kNeTO2dlg6ClpkzgZlVaV+qmH4eOLLGemcAZ9TJdpe+jFGSVIiIaRQtso9m5nYV6eOBb1BclPlOZp7dQ1afBa6reH8oRQcUiyluJ5Uk9VOOkCxJ6jYduAC4vDuh3pg1FBWFs6rWPx4YCzwADK1I3wq4IzO/HRHXAz9v1Q5IklaNlQNJEgCZeXv3SPUVlo1ZAxAR1wCHZuZZFK0My4mIfYBhwDbAcxExk6JS8a9ykZdaE70kqS9YOZAkNVJrzJrd6y2cmZ8HiIhJwOOZ+XJEfB84PyLeQfEswgoiYgowBWDUqFF9E7kkqdesHEiSGmlmzJoVF8icXjH9LHBCD8tPBaYCjBs3rsf8JUmt0cpxDiRJna+ZMWskSQOElQNJUiPLxqyJiLUpupye0eaYJEktYuVAkgRARFxNMSjlVhHRFREnZOZSirFnbgYeBK7LzPvbGackqXV85kCSBEBmHl0nfYUxaySpk1UPjOagaK+w5UCSJEkSYOVAkiRJUsnKgSRJkiTAyoEkSZKkkpUDSZIkSYCVA0mSJEkluzJtoLqbK0mSJGkgs3IwwNSq0Nh3ryRJkpph5UCSJEmDmhdXX+EzB5IkSZIAKweSJEmSSlYOJEmSJAFWDiRJkiSVrBxIkiRJAqwcSJIkSSrZlekgUN0912DtmkuSJEmN2XIgSZIkCWhx5SAixkfEQxExLyJOqTF/nYi4tpx/V0SMrpj3uTL9oYg4oNk8JUmSJK2cllUOImIIcCHwbmAb4OiI2KZqsROAf2TmFsDXgHPKdbcBJgLbAuOBb0bEkCbzlCRJkrQSWvnMwW7AvMycDxAR1wCHAg9ULHMo8OVy+nrggoiIMv2azHwB+
```
Comment (Higgins00) [inline Lab2.ipynb]: by a factor of 6 or 10^6?

**Comment by sierrawilde** [comment]
@oliviatessa I have finished lab 2 and uploaded the html version

**Comment by oliviatessa** [comment]
Hi Sierra, 

I'm sorry this is so late. Here are my comments for your lab 2. 

P.1 
- A Poisson convolved 20 times should actually be well approximated by a Gaussian. Try plotting them both on the same plot to see this comparison. 
- Try making plots more self-contained by adding captions (you can concisely repeat what you say in the written part of your report). Plots are one of the most important parts of a scientific document. Sometimes they are the only thing people read/look at

P.2 
- I'm glad you reintroduced the Chi-squared distribution, but try also introducing the problem statement a little more. This will make your report more essay-like and comprehensive. 

P.3
- Don't forget to interpret your results. Explain why it makes sense that the significance of your signal should decrease when you add more trials. 

P.4 
- It would be helpful to give a little detail (in words) about the equations you are writing in Latex. Briefly explain how/why you are using these. (To answer what questions?) 
- Why does the amount the brightness must increase decrease with number of trials? 

Best, 
Olivia

**Pull request closed without merging**